mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -24,21 +24,25 @@
 	</h2>
 
 	<div class="task-container">
-		<aui:input cssClass="input-item" name="task" />
+		<aui:input class="input-item" cssClass="input-item" name="task" placeholder="Enter your task.." />
```

Comment:
Why are you reverting my source formatting? In aui tags, use 'cssClass' instead of 'class', not in addition to 'class'

Also, use localization for all labels in UI.